.editorconfig

```diff
@@ -0,0 +1,10 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = false
```

Comment:
`true` would be nice, or are we trying to avoid too many incidental changes?

Reply:
Trying to avoid incidental changes for now, though Rubocop et al. should be doing this for us. Once the dust settles, we can update this.